release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -209,6 +208,10 @@ If you have already upgraded your cluster to {product-title} {product-version} a
 
 For more information, see xref:../networking/ovn_kubernetes_network_provider/converting-to-dual-stack.html#nw-dual-stack-convert_converting-to-dual-stack[Converting to a dual-stack cluster network] in _Converting to IPv4/IPv6 dual-stack networking_.
 
+[id="ocp-4-16-anp"]
+=== AdminNetworkPolicy is generally available
+This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Cluster Administrators can use ANP and BANP to apply cluster-scoped policies and safeguards for an entire cluster before namespaces are created. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale through multi-tenant isolation.
```

Comment:
```suggestion
This feature provides two new APIs, `AdminNetworkPolicy` (ANP) and `BaselineAdminNetworkPolicy` (BANP). Cluster Administrators can use ANP and BANP to apply cluster-scoped network policies and safeguards for an entire cluster before namespaces are created. Because it is cluster scoped, ANP provides Administrators a solution to manage the security of their network at scale without having to duplicate their network policies on each namespace.
```

Reply:
This is good. Thank you.